packages/common/tests/DateUtil.test.ts

```diff
@@ -0,0 +1,16 @@
+import { describe, expect, it } from 'vitest'
+import { DateUtil } from '../src/utils/DateUtil.js'
+
+const date = new Date()
+const year = new Date().getFullYear()
+const oneHourAgo = new Date(date.setHours(date.getHours() - 1)).toISOString()
+
+// -- Tests --------------------------------------------------------------------
+describe('DateUtil', () => {
```

Comment:
Not 100% sure about this, but by the time you declare `date` above, it might be out of sync with tests already. If it is the case you can use some of vitests tools to "mock" `Date` and set it to be a frozen value while they run.

Reply:
Mocked with `vi.setSystemTime`. Thanks 💯

- https://vitest.dev/api/vi#vi-setsystemtime